avm/src/daemon/linux.rs

```diff
@@ -65,13 +65,13 @@ async fn post_v1(endpoint: &str, body: Value) -> String {
   }
 }
 
-async fn post_v1_scale(cluster_id: &str, agz_b64: &str, deploy_token: &str, alan_version: &str, delta: i32) -> String {
+async fn post_v1_scale(cluster_id: &str, agz_b64: &str, deploy_token: &str, alan_version: &str, factor: &str) -> String {
```

Comment:
Why is this a string instead of a float64?

Reply:
I realized that it doesn't make a sense to parse it back and forth since the deploy service emits it and handles it so I'm just transmitting it as-is now from the daemon

Reply:
Oh, but doesn't it change the type when it is re-serialized back into JSON over the wire into a string type? This may only be working because of automatic type coercion in Javascript.

Reply:
the [endpoint](https://github.com/alantech/deploy/blob/1b3a83cad7cb0f1fa52de4c39f4b1d8ea3ef809c/src/routesV1.ts#L80) returns a string, `1` and not json :)

Reply:
![wtf](https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fmedia1.tenor.com%2Fimages%2Fa29634d2951380e2a5b6e71895367a3a%2Ftenor.gif%3Fitemid%3D7959057&f=1&nofb=1)